spec.bs

```diff
@@ -962,11 +961,11 @@ To <dfn>report result</dfn> given a [=leading bid info=] |leadingBidInfo|:
   1. Set |config| to |leadingBidInfo|'s [=leading bid info/auction config=].
   1. Let |sellerReportingScript| be the result of [=fetching script=] with |config|'s
     [=auction config/decision logic url=].
-  1. Let |sellerSignals| be the result of [=evaluating a reporting script=] with
-     |sellerReportingScript|, "`reportResult`", and
-     « |config|'s [=auction config/auction signals=], |browserSignals| ».
-  1. TODO: Set URLs passed to sendReportTo() into the FencedFrameConfig so they can be fetched when
-    ad renders.
+  1. Let « |sellerSignals|, |reportUrl| » be the result of [=evaluating a reporting script=] with
+    |sellerReportingScript|, "`reportResult`", and
+    « |config|'s [=auction config/auction signals=], |browserSignals| ».
+  1. If |reportUrl| is not null:
+    1. TODO: Set |reportUrl| into the FencedFrameConfig so they can be fetched when ad renders.

```

Comment:
Nit: Maybe link to `{{FencedFrameConfig}}` since that's the actual object you'll need to be using? If that doesn't work just let me know and I can fix it in our spec or something.

Reply:
it works!